src/hooks/useAccountProfile.ts

```diff
@@ -34,12 +34,12 @@ const buildAccountProfile = (
   );
 };
 
-export default function useAccountProfile() {
-  const accountProfileSelector = createSelector(
-    [walletSelector, settingsSelector],
-    buildAccountProfile
-  );
+const accountProfileSelector = createSelector(
```

Comment:
Took it out of the hook, because if the selector logic doesn't depend on any data from outside redux or from the components it is used in, there is no need to create multiple instances of that selector with reselect